api/app/views/mno_enterprise/jpi/v1/impac/dashboards/index.json.jbuilder

```diff
@@ -1 +1,2 @@
-json.array! @dashboards, partial: 'dashboard', as: :dashboard
\ No newline at end of file
+# Fully qualify template path to allow concern to be included in different modules
```

Comment:
@ouranos  We had to full qualify the template in order to reuse the DashboardController Concern in the TenantDashboard (See maestrano/opal-webstore#51) Do  you see problems with that ?

Reply:
The linked PR uses the non-admin concern in an admin controller. Not the best approach as the authorization logics are supposed to be different (not the case in this controller for now - see below)
So don't change the framework but the opal-webstore project